src/components/bridge/BridgeSelection.vue

```diff
@@ -224,6 +262,8 @@ export default defineComponent({
       buildEthereumBridgePageLink,
       stargateUrl,
       layerSwapLink,
+      zkatanaBridgeUrl,
```

Comment:
zKatana

Reply:
fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/1238/commits/73807824eb5f82f09b41027309290152d704db13) commit